client/my-sites/email/email-management/home/email-no-domain.jsx

```diff
@@ -5,37 +5,57 @@ import { isFreePlan } from '@automattic/calypso-products';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { useSelector } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import EmptyContent from 'calypso/components/empty-content';
+import { hasDomainCredit } from 'calypso/state/sites/plans/selectors';
 import Illustration from 'calypso/assets/images/customer-home/illustration--task-find-domain.svg';
 
 const EmailNoDomain = ( { selectedSite, translate } ) => {
+	const hasAvailableDomainCredit = useSelector( ( state ) =>
+		hasDomainCredit( state, selectedSite.ID )
+	);
+
 	if ( isFreePlan( selectedSite.plan.product_slug ) ) {
 		return (
 			<EmptyContent
 				action={ translate( 'Upgrade', { context: 'verb' } ) }
 				actionURL={ `/plans/${ selectedSite.slug }` }
 				illustration={ Illustration }
 				line={ translate(
-					'Upgrade now, claim your domain and pick from one of our flexible options to connect your domain with email and start getting emails today.'
+					'Upgrade now, set up your domain and pick from one of our flexible options to connect your domain with email and start getting emails today.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.